rust/src/delta_datafusion.rs

```diff
@@ -310,6 +448,7 @@ fn to_scalar_value(stat_val: &serde_json::Value) -> Option<datafusion::scalar::S
             }
         }
         serde_json::Value::String(s) => Some(ScalarValue::from(s.as_str())),
+        // TODO is it permissible to encode arrays / objects as partition values?
```

Comment:
not for deltalake: https://github.com/delta-io/delta/blob/master/PROTOCOL.md#partition-value-serialization